From 00b941f126852754ecb4b4150829011f245347d5 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Wed, 27 Jan 2016 10:58:30 +0800
Subject: [PATCH] Fix a haddock parse error

Before the change, I am getting the following error when running haddock:

```
src/Data/Bifunctor/TH.hs:675:5:
    parse error on input `-- * after substituting * for the dropped kind variables. If not, throw an error.'
```

Simply moving the first `*` to the end of previous line fixes this for me.
---
 src/Data/Bifunctor/TH.hs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Data/Bifunctor/TH.hs b/src/Data/Bifunctor/TH.hs
index 7a39bda..31a898c 100644
--- a/src/Data/Bifunctor/TH.hs
+++ b/src/Data/Bifunctor/TH.hs
@@ -671,8 +671,8 @@ buildTypeInstanceFromTys biClass tyConName dataCxt varTysOrig isDataFamily = do
         droppedTyVarNames :: [Name]
         droppedTyVarNames = concatMap tyVarNamesOfType droppedTysExpSubst
 
-    -- If any of the dropped types were polykinded, ensure that there are of kind
-    -- * after substituting * for the dropped kind variables. If not, throw an error.
+    -- If any of the dropped types were polykinded, ensure that there are of kind *
+    -- after substituting * for the dropped kind variables. If not, throw an error.
     unless (all hasKindStar droppedTysExpSubst) $
       derivingKindError biClass tyConName
 
